### PR TITLE
Add Users and Sections Models

### PR DESCRIPTION
**Pull request by MihirK1212**

### CODE REVIEW

File: server/models/users.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    sectionID : Number,
+    sectionHeader : String,
+    sectionContent: {
+        sectionChildName : String,
+        sectionChildImage : String,
+        sectionChildDesc : String
+    }
+})
+
+const clubsSchema = mongoose.Schema({
```
Comment (akashg3627) [inline server/models/users.js:14]: don't use words like club as we are making it general
you can use User instead

Comment (MihirK1212) [reply server/models/users.js:14]: everything else is ok ?

File: server/models/users.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import mongoose from 'mongoose';
+
+const sectionsSchema = mongoose.Schema({
+    sectionName : String,
+    sectionID : Number,
+    sectionHeader : String,
+    sectionContent: {
```
Comment (akashg3627) [inline server/models/users.js:7]: This will be array of objects

File: server/models/users.js (hunk 1 above)
Comment (akashg3627) [reply server/models/users.js:14]: Also, use seperate file for both model

File: server/models/users.js (hunk 2 above)
Comment (MihirK1212) [reply server/models/users.js:7]: bhaiya but i am creating a separate schema for it so it is already an array of records right ?

**Review by akashg3627 — APPROVED** [review]

**Pull request merged**